specs-go/config.go

```diff
@@ -679,7 +680,7 @@ type LinuxSyscall struct {
 }
 
 // LinuxIntelRdt has container runtime resource constraints for Intel RDT
-// CAT and MBA features which introduced in Linux 4.10 and 4.12 kernel
+// CAT, MBA, CMT and MBM features which introduced in Linux 4.10 and 4.12 kernel
```

Comment:
How about:
"// LinuxIntelRdt has container runtime resource constraints and monitoring metrics for Intel RDT CAT, MBA, CMT and MBM features which are available in Linux 4.14 and newer kernel versions"?